### PR TITLE
Adding note

### PR DESCRIPTION
**Pull request by get-itips**
Closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/6362

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/manage-resource-accounts.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ If you're planning to use the resource account with an auto attendant or call qu
 
 4. Click **Save**.
 
+> [!NOTE] To see a service number listed in the **Assigned phone number** box, buy a Phone System or get a Phone System - Virtual user license and make sure it's not assigned to any other voice services. [Learn more](https://docs.microsoft.com/microsoftteams/office-365-licensing?WT.mc_id=TeamsAdminCenterCSH). If the service number is still not listed in the dropdown, use the [Set-CsOnlineVoiceApplicationInstance](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoiceapplicationinstance) cmdlet to assign the number to the resource account.
```
Comment (JohanFreelancer9) [inline Teams/manage-resource-accounts.md:99]:
```suggestion
> [!NOTE]
> To see a service number listed in the **Assigned phone number** box, buy a Phone System or get a Phone System Virtual User license and make sure it's not assigned to any other voice services. To learn more, see [Microsoft Teams service description](https://docs.microsoft.com/microsoftteams/office-365-licensing?WT.mc_id=TeamsAdminCenterCSH). If the service number is still not listed in the dropdown, use the [Set-CsOnlineVoiceApplicationInstance](https://docs.microsoft.com/powershell/module/skype/set-csonlinevoiceapplicationinstance) cmdlet to assign the number to the resource account.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by MikePlumleyMSFT** [comment]
This is too much of an edge case to include in the core docs.

**Pull request closed without merging**